lib/slack.py

```diff
@@ -104,30 +111,51 @@ def __build_message__(self, suggested_video, channel, username, search_id, index
 
         return params
 
+    def __get_videos__(self, search_id):
```

Comment:
Sidenote: We should be meaningful in the way we order our functions (probably best to put the public ones first e.g. suggest_video and the private ones after)